frontend/pendingTranslations.ftl

```diff
@@ -137,6 +137,7 @@ phone-onboarding-step3-code-success-body = You’re now ready to choose your pho
 phone-onboarding-step3-code-success-subhead-title = What’s next?
 phone-onboarding-step3-code-success-subhead-body = Choose your phone number mask and start using { -brand-name-firefox-relay } to protect your true phone number.
 phone-onboarding-step3-code-success-cta = Search for phone number masks
+phone-onboarding-step3-code-success-cta-2 = Search for masks
```

Comment:
Don't forget to submit this to the l10n repo (in the `phones.ftl` file).

Reply:
PR opened here, since @lloan is out.

Reply:
@Vinnl: Is it okay to leave this in pending for the time being?

Reply:
Thank you! And yes, if the string is (eventually) present in both the l10n repo and pending, the former will have precedence.